app/ide-desktop/client/tests/localWorkflow.spec.ts

```diff
@@ -60,22 +80,15 @@ electronTest('Local Workflow', async ({ page, app, projectsDir }) => {
   await expect(writeNode.locator('.TableVisualization')).toContainText('output_ensodryrun')
 
   expect(await fs.readdir(PROJECT_PATH)).not.toContain(OUTPUT_FILE)
-
+  await expect(page.locator('.GraphEditor .GraphNode.pending')).toHaveCount(0)
   // Press `Write once` button.
   await writeNode.locator('.More').click()
   await writeNode.getByTestId('recompute').click()
+  await page.mouse.move(0, 0) // Avoid showing a tooltip
+  await expect(page.locator('.GraphEditor .GraphNode.pending')).toHaveCount(0)
```

Comment:
This check may pass before finishing execution, because we may run this check before the first "pending" updates arrive from the engine.

We may add expectation to have _some_ node non-pending first, but I'm not sure how it would behave in really fast computations.

Reply:
There are cases when pending status don't show up at all, so I don't know how to adjust that. This check did stabilize the cases where nodes were still pending for a split second after the run, but I didn't see the inverse problem in practice.